cluster/addons/rbac/cluster-autoscaler/cluster-autoscaler-rbac.yaml

```diff
@@ -6,6 +6,14 @@ metadata:
     addonmanager.kubernetes.io/mode: Reconcile
 rules:
   # leader election
+  - apiGroups: ["coordination.k8s.io"]
+    resources: ["leases"]
+    verbs: ["create"]
+  - apiGroups: ["coordination.k8s.io"]
+    resources: ["leases"]
+    resourceNames: ["cluster-autoscaler"]
+    verbs: ["get", "update", "patch", "delete"]
+  # TODO: remove N releases after upgrading cluster autoscaler to a version that uses leases
```

Comment:
can you add a name or specific release to this TODO? as-is, this isn't very actionable